tests/intl/intltest.cpp

```diff
@@ -417,11 +417,35 @@ TEST_CASE("wxUILocale::IsSupported", "[uilocale]")
 
 TEST_CASE("wxUILocale::GetInfo", "[uilocale]")
 {
-    CHECK( wxUILocale::FromTag("en").GetInfo(wxLOCALE_DECIMAL_POINT) == "." );
+    const wxUILocale locEN(wxUILocale::FromTag("en-US"));
+    CHECK( locEN.GetInfo(wxLOCALE_DECIMAL_POINT) == "." );
+    CHECK( locEN.GetCurrencySymbol() == "$");
+    CHECK( locEN.GetCurrencyCode() == "USD");
+    CHECK( locEN.GetCurrencyInfo().CurrencyFormat.FractionalDigits == 2);
+    CHECK( locEN.GetCurrencySymbolPosition() == wxCurrencySymbolPosition::PrefixNoSep);
+    CHECK( locEN.UsesMetricSystem() == wxMeasurementSystem::NonMetric);
+
+    const wxUILocale locDE(wxUILocale::FromTag("de-DE"));
+    if (CheckSupported(locDE, "German"))
+    {
+        CHECK( locDE.GetInfo(wxLOCALE_DECIMAL_POINT) == ",");
+        CHECK( locDE.GetCurrencySymbol() == L"\u20AC");
```

Comment:
I'm almost sure that using
```suggestion
        CHECK( locDE.GetCurrencySymbol() == L"€");
```
should work everywhere nowadays.